### DISCUSSION TITLE
Sitemap

### DISCUSSION THREAD

**Discussion by elyserobinson** [adityatelange/hugo-PaperMod#768] [Q&A]
Which file do you use for the sitemap (index or sitemap)? When I add index.xml to Google News it is pulling the wrong image even though it has the right image in the code. Trying to figure out why Google News is pulling the wrong image. Or should I use sitemap.xml?

**Comment by leyiwu** [comment]
You may add 

sitemap:
  changefreq: weekly
  priority: 0.8
title: News


to your config.yml file, which would generate a sitemap.xml file for you. You can also override it. Reference: Hugo docs https://gohugo.io/templates/sitemap-template/
